### ISSUE TITLE
[Craft] extension doesn't work with Setapp Craft installation

### ISSUE THREAD

**Issue by 0x77dev** [raycast/extensions#4241]
### Extension

https://www.raycast.com/bgnfu7re/craftdocs

### Description

<img width="1137" alt="Screenshot 2023-01-13 at 00 57 02" src="https://user-images.githubusercontent.com/46429701/212205983-3a937bc6-00de-4d1f-9d44-fa2988127918.png">

### Steps To Reproduce

1. Install [Craft from Setapp](https://setapp.com/apps/craft)
2. Sign in inside Craft and create a document or daily note
3. Install [Craft Extension for Raycast](https://www.raycast.com/bgnfu7re/craftdocs) 
4. Try searching blocks or opening daily notes, and it will show `Craft not initialized`.

### Current Behaviour

When installed with Setapp it seems that the Craft app stores in different data path not as specified here:

https://github.com/raycast/extensions/blob/7ecfadbd3c5e81398a4ebbe66f4985ee1779a735/extensions/craftdocs/src/Config.ts#L5

### Expected Behaviour

Extension to function correctly and recognize Craft when installed using Setapp.

**Comment by 0x77dev** [comment]
The Setapp version of Craft uses different bundle identifier:

`Info.plist`

```xml
<key>CFBundleIdentifier</key>
<string>com.lukilabs.lukiapp-setapp</string>
<key>CFBundleInfoDictionaryVersion</key>
<string>6.0</string>
<key>CFBundleName</key>
<string>Craft</string>
```

**Comment by 0x77dev** [comment]
The data path is different because of different `CFBundleIdentifier`

![terminal screenshot](https://user-images.githubusercontent.com/46429701/212207295-a314eef1-9310-430b-a69a-404c392640b2.png)

**Comment by 0x77dev** [comment]
I crated a https://github.com/raycast/extensions/pull/4243 with a fix.

**Comment by 0x77dev** [comment]
Released #4243
